xml/System.Data.Objects/ObjectParameterCollection.xml

```diff
@@ -200,7 +200,7 @@
         <Parameter Name="name" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="name">The name of the parameter to look for in the collection. This name should not include the "@" parameter marker that is used in the [!INCLUDE[esql](~/includes/esql-md.md)] statements, only the actual name.</param>
+        <param name="name">The name of the parameter to look for in the collection. This name should not include the "\@" parameter marker that is used in the [!INCLUDE[esql](~/includes/esql-md.md)] statements, only the actual name.</param>
```

Comment:
undo the change to escape the @ here and instead replace the include entry with Entity SQL